pwndbg/disasm/__init__.py

```diff
@@ -83,8 +83,16 @@ def get_disassembler(pc):
     extra = None
 
     if pwndbg.arch.current in ('arm', 'aarch64'):
-        extra = {0:CS_MODE_ARM,
-                 0x20:CS_MODE_THUMB}[pwndbg.regs.cpsr & 0x20]
+        if pwndbg.regs.xpsr:
+            extra = {
+                0: CS_MODE_MCLASS,
+                0x1000000: CS_MODE_MCLASS | CS_MODE_THUMB,
```

Comment:
https://github.com/aquynh/capstone/blob/d70e2b286a7466ab8ed6895213acc24e52b8b2fa/bindings/powershell/Capstone/Capstone.psm1#L120 nice

Reply:
As for this, I'm not sure the flags should be `CS_MODE_MCLASS | CS_MODE_THUMB` or just `CS_MODE_THUMB`. Not yet look into their codes.